src/control/lib/daos/system_prop_test.go

```diff
@@ -177,3 +178,21 @@ func TestDaos_SystemPropertyKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestDaos_SystemPropertySelfHealHasFlag(t *testing.T) {
```

Comment:
I lean toward using a table-based test for this type of thing, to make it a little simpler to add/remove cases. I wouldn't block on that though.